lib/puppet/type/scheduled_task.rb

```diff
@@ -83,6 +83,15 @@ def insync?(current)
       to determine if a scheduled task is in sync or not."
   end
 
+  newproperty(:compatibility) do
+    desc "The compatibility level associated with the task. May currently only
```

Comment:
Remove `May`  It's not optional

Reply:
Well, specifying the value *is* optional to specify (has to be for backwards compatibility)... but specifying anything other than `1` is not optional at the moment.

Note that this will change / be updated in a later commit as v2 is fleshed out.